File: services/utils-service/src/main/java/com/sgpttt/UtilsService/model/Career.java
package com.sgpttt.UtilsService.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

@Entity
public class Career {

    @Id
    @GeneratedValue(strategy=GenerationType.AUTO)
    private Long career_id;

    private String name;

    // Getters and Setters
    public Long getCareer_id() {
        return career_id;
    }
    public void setCareer_id(Long id) {
        this.career_id = id;
    }

    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
}

File: services/protocols-service/src/main/kotlin/com/sgptt/protocolsservice/repository/entity/Career.kt
package com.sgptt.protocolsservice.repository.entity

import com.sgptt.protocolsservice.model.Career
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.EnumType
import jakarta.persistence.Enumerated
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "career")
data class Career(
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	@Column(name = "career_id")
	val careerId: Long,
	
	@Enumerated(EnumType.STRING)
	@Column(name = "name", columnDefinition = "varchar(50)", nullable = false)
	val career: Career
)
